framework/validators/FileValidator.php

```diff
@@ -80,6 +80,15 @@ class FileValidator extends Validator
      * @see tooMany for the customized message when too many files are uploaded.
      */
     public $maxFiles = 1;
+
+    /**
+     * @var int the minimum file count the given attribute can hold.
+     * Defaults to 1, meaning single file upload. By defining a higher number,
```

Comment:
Defaults to 0. Higher value means at least that number of files should be uploaded.

Reply:
I fixed it